netbox/extras/scripts.py

```diff
@@ -399,25 +434,54 @@ def as_form(self, data=None, files=None, initial=None):
 
     # Logging
 
-    def log_debug(self, message):
-        self.logger.log(logging.DEBUG, message)
-        self.log.append((LogLevelChoices.LOG_DEFAULT, str(message)))
+    def _log(self, message, obj=None, log_level=LogLevelChoices.LOG_DEFAULT, level=logging.INFO):
```

Comment:
It's very confusing to pass both `log_level` and `level`. Can't we infer one from the other?

Reply:
For legacy reports log function (just plain log not log_debug or log_exception) if we change that from going to [logger.info](http://logger.info/) to logger.debug then I can make a map.  Right now both log and log_debug use LogLevelChoices.LOG_DEFAULT but log goes to info and log_debug goes to debug so there isn't a 1:1 mapping, but if we can change this then can make one map to the other.